src/Robo/Commands/Git/GitCommand.php

```diff
@@ -47,13 +47,16 @@ public function commitMsgHook($message) {
    * @return int
    */
   public function preCommitHook($changed_files) {
+    $changed_files_list = explode("\n", $changed_files);
     $this->invokeCommands([
+      // Passing a file list to be PHPCS will cause all specified files to
```

Comment:
We don't need to add this comment back :)

Reply:
I refactored and removed all the changes to this class.